ci/tests/e2e-ansible.sh

```diff
@@ -44,8 +44,8 @@ remove_operator() {
     kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/service_account.yaml"
     kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/role.yaml"
     kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/role_binding.yaml"
-    kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/crds/ansible.example.com_memcacheds_crd.yaml"
-    kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/crds/ansible.example.com_foos_crd.yaml"
+    timeout 60s bash -c -- "kubectl delete --wait=true --ignore-not-found=true -f \"$OPERATORDIR/deploy/crds/ansible.example.com_memcacheds_crd.yaml\""
```

Comment:
I think for these, we can use `kubectl`'s `--timeout` flag rather than wrapping the bash timeout command.

Reply:
I agree, just used bash initially since that's what came to mind. Re: why these are failing, I need to take a deeper dive, just trying to shorten the time to wait to fail right now